src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -199,6 +200,14 @@ public void onKill(EntityDeathEvent e) {
             }
         }
 
+        if (entity instanceof Steerable) {
+            Steerable steerableEntity = (Steerable) entity;
+
+            if (steerableEntity.hasSaddle()) {
+                items.remove(new ItemStack(Material.SADDLE));
```

Comment:
I'm not sure this will always work. You're making a new object rather than getting the existing which means it NEEDS to match this.

What happens if the saddle is renamed? What if it has a custom model data?

We should probably loop the items and remove if type == SADDLE instead

Reply:
But when you kill a pig with for example renamed saddle then pig will drop you a normal saddle without name, lore, tags etc.